.vscode/settings.json

```diff
@@ -1,8 +1,10 @@
 {
+	"root": true,
```

Comment:
![image](https://github.com/huggingface/chat-ui/assets/342922/a7884161-205f-4404-bce5-7cd5f887c396)

Seems VSCode doesn't like it? It's unused